Assets/MXR.SDK/Runtime/Types/HomeScreenTypes.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+
+namespace MXR.SDK {
+    /// <summary>
+    /// Represents the current state of the homescreen.
+    /// This state is sent to the MXR Web Dashboard via
+    /// the MXR Admin App for reporting.
+    /// </summary>
+    [Serializable]
+    public class HomeScreenState {
+        /// <summary>
+        /// The current view of the homescreen
+        /// </summary>
+        public HomeScreenView view;
+
+        /// <summary>
+        /// The details associated with the view.
+        /// Currently the SDK supports video related details.
+        /// </summary>
+        public HomeScreenViewDetails viewDetails = new HomeScreenViewDetails();
+    }
+
+    /// <summary>
+    /// The different view types recognised by the SDK. 
+    /// </summary>
+    [Serializable]
+    public enum HomeScreenView {
+        LIBRARY,
+        VIDEO_PLAYER,
+        WIFI,
+        ADMIN_PIN,
+        ADMIN_SETTINGS,
+        SHORTCUT_MENU
+    }
+
+    /// <summary>
+    /// Details associated with the view of the home screen.
+    /// Currently contains fields for video reporting.
+    /// </summary>
+    [Serializable]
+    public class HomeScreenViewDetails {
+        /// <summary>
+        /// The ID of the video currently being played
+        /// </summary>
+        public string videoId;
+
+        /// <summary>
+        /// Where in the video is the viewer (in seconds)
+        /// </summary>
+        public int videoLocation;
+
+        /// <summary>
+        /// The duration of the video currently being played (in seconds)
```

Comment:
Just confirming compared to the spec - is the choice of seconds here intentional? Do we require seconds, instead of milliseconds?

Reply:
This is wrong, the values are supposed to be in milliseconds. Should also change from type `int` to `long` to accommodate larger values.

Reply:
fixed in 07480bbf794b3eeca9d4d4c6a88b9f18b6d89fa7. The type is `long` now as values can be much larger.